Alerting: add docs to run HA on kubernetes (#42877)
grafana/grafana: docs/sources/alerting/unified-alerting/high-availability.md

    fieldRef:
      fieldPath: status.podIP
```

You can also connect the different Grafana pods by using a headless service and DNS service discovery.

1. Create a headless service for you Grafana deployment.

> 2. Change you grafana.ini so that the instances running in the pods are using this service for service discovery:

Reviewer: ```suggestion
2. Modify the `grafana.ini` file so that the instances running in the pods are using this service for service discovery:
```

   ```toml
   ...
   [unified_alerting]
     enabled = true
     ha_peers = "grafana-headless-service:9094"
   [alerting]
     enabled = false
   ```

> 3. Check your logs to see if the instances are connecting to each other. You should see one of those messages for every server:

Reviewer: ```suggestion
3. Check the logs and verify that the instances are connecting to each other. You should see one of those messages for every server:
```
